spack/packages/dla-future/package.py

```diff
@@ -25,11 +25,15 @@ class DlaFuture(CMakePackage, CudaPackage, ROCmPackage):
 
     variant("miniapps", default=False, description="Build miniapps.")
 
+    variant("c_api", default=False, description="Build C API")
```

Comment:
I don't feel strongly about the name of this variant, but I had a quick look in spack and found:
- `variant("c", ...)`: 8 occurrences
- `variant("c_interface", ...)`: 1 occurrence
- `variant("c_shared", ...)`: 1 occurrence (not quite the same)

There were also 15 occurrences of `variant("cxx", ...)`. What do you think about making it
```suggestion
    variant("c", default=False, description="Build C API")
```
?